static/js/admin/src/action/ExportMetadataXMLBatchAction.js

```diff
@@ -71,7 +68,7 @@ define('ExportMetadataXMLBatchAction', [ 'jquery', 'AbstractBatchAction', "tpl!.
 			}
 			
 			$.ajax({
-				url : includeChildren? "exportContainerXML" : "exportXML",
```

Comment:
I think this will need to go to the services application now, which probably just means switching to "/services/exportXML"

Reply:
fixed